app/routes.py

```diff
@@ -16,6 +18,26 @@ def checker():
     return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
 
 
+@app.route('/checker/ready', methods=['GET'])
+def readiness():
+    sphinx_ok_string = 'READY\n'
+    try:
+        with open(SPHINX_BACKEND_READY, 'r', encoding='utf-8') as fd:
+            content = fd.read()
+    except IOError:
+        content = ""
```

Comment:
I would log the error here for debugging and might be a good idea to use a critical level.